android/app/src/main/java/app/organicmaps/sound/TtsPlayer.java

```diff
@@ -198,11 +215,10 @@ public void speak(String textToSpeak)
     if (Config.TTS.isEnabled())
       try
       {
-        boolean isMusicActive = mAudioFocusManager.requestAudioFocus();
-        if (isMusicActive)
-          delayHandler.postDelayed(() -> mTts.speak(textToSpeak, TextToSpeech.QUEUE_ADD, mParams, textToSpeak), TTS_SPEAK_DELAY_MILLIS);
-        else
-          mTts.speak(textToSpeak, TextToSpeech.QUEUE_ADD, mParams, textToSpeak);
+        final boolean isMusicActive = mAudioManager.isMusicActive();
+        AudioManagerCompat.requestAudioFocus(mAudioManager, mAudioFocusRequest);
+        final long delay = isMusicActive ? TTS_SPEAK_DELAY_MILLIS : 0;
```

Comment:
What does git blame/git log say about this delay when music is active? Why it was introduced in the first place? Is this delay really needed?

Reply:
That delay was introduced in this 2021 commit: https://github.com/organicmaps/organicmaps/commit/959dc0834a4eb576968678b440d91d90d93fa148